tap_mailchimp_export/schemas/automation_workflows.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "object",
+  "properties": {
```

Comment:
once you lock down automation workflows, can you also build out all the schemas properly